lib/workers/repository/process/lookup/index.ts

```diff
@@ -384,6 +384,24 @@ export async function lookupUpdates(
           // TODO #7154
           update.newDigest =
             update.newDigest ?? (await getDigest(config, update.newValue))!;
+
+          // If the digest could not be determined, report this as otherwise the
+          // update will be omitted later on without notice.
+          if (update.newDigest === null) {
+            logger.debug(
+              { depName, ...update },
```

Comment:
How verbose is this debug message?

Reply:
I've trimmed it a little. I think most of these fields are not strictly needed.